src/Frontend/Core/Engine/TwigTemplate.php

```diff
@@ -26,31 +22,17 @@ class TwigTemplate extends BaseTwigTemplate
     private $themePath;
 
     public function __construct(
-        Twig_Environment $environment,
+        Environment $environment,
         TemplateNameParserInterface $parser,
         FileLocatorInterface $locator
     ) {
-        $container = Model::getContainer();
-        $this->forkSettings = $container->get('fork.settings');
         $this->language = Locale::frontendLanguage();
 
-        parent::__construct(clone $environment, $parser, clone $locator);
+        parent::__construct(clone $environment, $parser, clone $locator, 'Frontend');
```

Comment:
Can't `'Frontend'` be replaced by `APPLICATION`?

Reply:
No, cause then you won't be able to use it correctly in a console command